### ISSUE TITLE
Password Sharing 

### ISSUE THREAD

**Issue by TriplEight** [Rabbit-Company/Passky-Desktop#13]
**Is your feature request related to a problem? Please describe.**
Almost all the modern password managers have the feature of password sharing. Except of offline ones.

I get it - it's great when I want to move from another password manager. But with Passky I can only share my entire db, why would I need that If I want to share just one password.

**Describe the solution you'd like**
Either with PSK like Enpass does it: produce an encrypted binary and share a file by any mean of communication.
Or RBAC, like in 1pass or Passbolt, that would work great for teams: share passwords or vaults with users and groups. But that, for sure, would require a rework of UI and might go against your minimalism principle.

**Describe alternatives you've considered**
Basically I can't use Passky with a team without this feature. Maybe only consider for a personal use, but still - can't share with family.
